pkg/grpc/device_definition.proto

```diff
@@ -260,6 +262,21 @@ message CreateDeviceStyleResponse {
 	string ID = 1;
 }
 
+message GetDeviceCompatibilityListRequest {
+	string makeId = 1;
+}
+
+message DeviceCompaitibilityList {
+	string model = 1;
+	int32 year = 2;
+	google.protobuf.Struct features = 3;
+}
+
+message GetDeviceCompatibilityListResponse {
+	string make = 1;
+	repeated DeviceCompaitibilityList deviceCompatilities = 2;
```

Comment:
I know we had a mix of camelCase and lower_snake_case, but if you see the latest PR we're trying to keep everything standard to `lower_snake_case`
```suggestion
	repeated DeviceCompaitibilityList device_compatilities = 2;
```